core-services/prow/02_config/_config.yaml

```diff
@@ -286,7 +287,12 @@ plank:
       timeout: 23h0m0s
     repo: openshift/distributed-tracing-qe
   - config:
-      timeout: 12h0m0s
+      gcs_configuration:
+        bucket: qe-private-deck
+        mediaTypes:
+          log: text/plain
+      gcs_credentials_secret: gce-sa-credentials-gcs-qe-private-deck
+      timeout: 23h0m0s
```

Comment:
this part doesn't need to change

Reply:
do you mean only **timeout** or the whole section? @smg247

Reply:
just the timeout